### DISCUSSION TITLE
~/whynot

### DISCUSSION THREAD

**Discussion by James-MF-Thomas** [UltimaHoarder/UltimaScraper#656] [Q&A] [answered]
So the cmd line (macOS) and python: "cd ~/Documents" takes you to the user docs folder,  

In the OFscript: " ~/arbitrary " will not register as a location/destination.

just curious why.

**Comment by M-rcus** [comment, marked as answer]
`~` = "home folder" is a feature of the shell (known as "tilde expansion").

This software uses Python and thus a different method of resolving paths (specifically [`os.path`](https://docs.python.org/3/library/os.path.html) I believe).
